### DISCUSSION TITLE
help installing last asp.net core version - demo modules not found

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#7145] [Help]
Hi,

I'm trying to install the latest version using VS 2022 with the demo modules but I get the error below. How can I fix it?

![image](https://github.com/serenity-is/Serenity/assets/17210232/76b1fb67-3bf5-4ab8-a33d-e4b0645832aa)

**Comment by VictorTomaili** [comment]
For the latest pro features, you need to purchase startsharp. 
ensure the .net 8 installed on your system.

**Comment by teafmartins** [comment]
Hi @VictorTomaili 
Yes, I have it, I'm using startsharp.
.net 8 is also installed
Any idea what should I check?

**Reply by VictorTomaili** [reply]
You can ask at the StartSharp repository. Is this a new project, or are you attempting to upgrade an existing one?

**Reply by teafmartins** [reply]
@VictorTomaili New one with demo modules.

**Reply by VictorTomaili** [reply]
Clear your NuGet cache by executing `nuget locals all -clear` and check the nuget.config for errors, which can be found in the project directory.
